worlds/generic/docs/setup_en.md

```diff
@@ -1,18 +1,21 @@
 # Archipelago Setup Guide
 
-This guide is intended to provide an overview of how to install, set up, and run the Archipelago multiworld software.
-This guide should take about 5 minutes to read.
+This guide is intended to provide an overview of how to install, set up, and run the Archipelago multiworld software, as
+well as how to generate and host multiworlds, and how to connect to the multiworld after hosting has begun. This is a
+general overview; for more specific steps, reference the relevant game's [setup guide](/tutorial).
 
 ## Installing the Archipelago software
 
-The most recent public release of Archipelago can be found on the GitHub Releases page. GitHub Releases
-page: [Archipelago Releases Page](https://github.com/ArchipelagoMW/Archipelago/releases).
+The most recent public release of Archipelago can be found on the GitHub Releases page:
+[Archipelago Releases Page](https://github.com/ArchipelagoMW/Archipelago/releases).
 
 Run the exe file, and after accepting the license agreement you will be prompted on which components you would like to
```

Comment:
"Prompted on" isn't usually used in this way, changing it to "...you will be asked which components..." would be better